nxt/builtin/sub_graphs.nxt

```diff
@@ -34,8 +34,10 @@
                 "            continue",
                 "        attr_path = nxt_path.make_attr_path(nxt_path.WORLD, attr_name)",
                 "        params[attr_path] = attr_val",
-                "    ",
-                "    result_rt_layer = sub_stage.execute(parameters=params, start=self._start)",
+                "    try:",
+                "        result_rt_layer = sub_stage.execute(parameters=params, start=self._start)",
+                "    except ExitGraph as exit_graph:",
+                "        result_rt_layer = exit_graph.runtime_layer",
```

Comment:
A bit picky, but can we get a simple test for this too?

Reply:
Shoot, I said I'd do that and never did. I'll whip up a test.